core/server/services/email-analytics/lib/event-processor.js

```diff
@@ -136,10 +137,30 @@ class GhostEventProcessor extends EventProcessor {
             return false;
         }
 
+        const subscribedNewsletterIds = (
```

Comment:
Not a big deal - this can be slightly simplified to
```js
const subscribedNewsletterIds = await this.db.knex('members_newsletters')
    .where('member_id', '=', memberId)
    .pluck('newsletter_id');
```